.gitignore

```diff
@@ -7,5 +7,6 @@ build
 Gemfile
 native_build
 *.gdb
+.venv
```

Comment:
@stephanie-eng adding to gitignore since following the new instructions will create this folder locally

Reply:
Good point, thanks for the fix